src/main.cpp

```diff
@@ -27,15 +27,16 @@
 #include "posture_estimator.h"
 
 #define NUM_LOOPS 500
-#define NUM_INF_CORE_THREADS 8
+#define NUM_INF_CORE_THREADS 16
```

Comment:
I think we should probably keep this at 8 for now, in a separate issue we can look into choosing the best value